### DISCUSSION TITLE
Why does photoprism rename the file/title after I have changed location?

### DISCUSSION THREAD

**Discussion by tobivan** [photoprism/photoprism#4411] [Q&A]
Hi there,

I am trying a docker installation of photoprism. For first tests there are folders that I have included like this
/volume1/mypictures:/photoprism/originals

Folders show up under "Folders" as expected, that's fine so far. Then I have tried to change location of some pictures, which seems to be only possible one after the other. So I clicked on the edit-button and added the GPS e.g. 49.123123123123, 12.123123123123
Works fine, too, BUT suddenly the name of the picture is no longer its original "file name" but it is renamed according to the place of the GPS and with a prefix of integrated AI e.g. "Building / Munich / 2020"

How can I prevent that, I want to keep the original file name?

Thanks a lot!
Tobias

**Comment by graciousgrey** [comment]
Hey :wave: 

The title is generated automatically by PhotoPrism, e.g. based on the location, labels or people recognized (unless a title is already defined in the metadata of the files). 
There is currently no option to prevent PhotoPrism from generating titles, but you can edit them manually. 
Please note that the file names of the files in your Originals folder will not be changed. Batch editing is also not yet available, but planned for this year :)

**Reply by lastzero** [reply]
It would still be interesting to know what the title was before if it was based on the filename? Maybe we can simply give it a higher priority if it was good:
- https://github.com/photoprism/photoprism/blob/develop/internal/entity/src.go

**Reply by tobivan** [reply]
Thanks. That's a great idea, higher priority!
The title is exactly the filename without its extension .jpg

**Reply by lastzero** [reply]
Can you give one or two specific examples of filenames and the titles generated, as this depends on the naming pattern? Not all filenames are suitable for generating titles, e.g. `IMG_1234.jpg` would be ignored and titles generated from folder names should probably also have a lower priority as the title is then the same for all pictures in that folder, so not very useful.

**Reply by tobivan** [reply]
Meanwhile I have named "People" and things are getting clearer.
It seems as if my first statement was not right: the title within photoprism is not being generated according to its filename but according to some "AI-recognition" and now with people's names, too. Even without new GPS coordinates the title is being changed, e.g. a file called "threema-20231108-83635.jpg" now has the title "Building / Munich / 2023" or "Eva / Outdoor / 2013".
The software seems to be "rather intelligent" :-) Maybe I should sort out its features first. Thing is I am used to finding old pictures by their filename and not by an intelligent shelf-thing :-)

**Reply by wsyxbcl** [reply]
I handle this by editing the 'title' field in the image metadata and then update, the title can be recognized by Photoprism and won't be overwritten unless manually changed.
